records_mover/session.py

```diff
@@ -36,11 +37,26 @@ def _infer_session_type() -> str:
 
 
 def _infer_scratch_s3_url(session_type: str) -> Optional[str]:
-    scratch_s3_url: Optional[str]
-
     if "SCRATCH_S3_URL" in os.environ:
         return os.environ["SCRATCH_S3_URL"]
 
+    # config_resolver logs at the WARNING level for each time it
+    # attempts to load a config file and doesn't find it - which given
+    # it searches a variety of places, is quite noisy.
+    #
+    # https://github.com/exhuma/config_resolver/blob/master/doc/intro.rst#logging
+    #
+    # https://github.com/exhuma/config_resolver/issues/69
+    logging.getLogger('config_resolver').setLevel(logging.ERROR)
+    config_result = get_config('records_mover', 'bluelabs')
+    cfg = config_result.config
+    if 'aws' in cfg:
+        s3_scratch_url: Optional[str] = cfg['aws'].get('s3_scratch_url')
+        if s3_scratch_url is not None:
+            return s3_scratch_url
+    else:
+        logger.debug('No config ini file found')
+
     if session_type == 'cli':
         try:
```

Comment:
There will be a second PR to retire this feature once we have internal uses switched over to the config file.